app/utils/track.js

```diff
@@ -145,7 +145,7 @@ export function track(params, usePrefix = true, force = false) {
   const isNonInteractive = params.isNonInteractive !== false;
   const pvUID = window.pvUID;
   const {
-    action, label = '', value = 0, trackingMethod = TrackingMethod.both,
+    action, label = '', value = 0, trackingMethod = TrackingMethod.internal,
```

Comment:
Oh, this looks wider I thought the change would be 🙈 It means all events on mobile-wiki will get to DW instead of DW&GA. I thought the change will be connected to `mercury-scroll_speed`. You mentioned you tested it with GTM, thought. When was the change?

Reply:
My proposal is to leave it as-is and pass `trackingMethod: TrackingMethod.internal` in https://github.com/Wikia/mobile-wiki/pull/2292/files#diff-b109f7157f457fb456191851d60646c0R220-R224

Reply:
Oh, weird... I think I got my files confused. Guess that's why we do PRs, right. Thanks nAndy, I'll get it fixed over the weekend.